src/hardware/sblaster.cpp

```diff
@@ -294,6 +308,136 @@ static void InitializeSpeakerState()
 	}
 }
 
+
+static const std::map<SB_TYPES, FILTER_TYPES> sb_type_to_filter_type_map = {
+        {SBT_NONE, FT_NONE},
+        {SBT_1, FT_SB1},
+        {SBT_2, FT_SB2},
+        {SBT_PRO1, FT_SBPRO1},
+        {SBT_PRO2, FT_SBPRO2},
+        {SBT_16, FT_SB16},
+        {SBT_GB, FT_NONE},
+};
+
+static const std::map<std::string, FILTER_TYPES> filter_map = {
+        {"none", FT_NONE},
+        {"sb1", FT_SB1},
+        {"sb2", FT_SB2},
+        {"sbpro1", FT_SBPRO1},
+        {"sbpro2", FT_SBPRO2},
+        {"sb16", FT_SB16},
+};
```

Comment:
These are only used inside `configure_filters`'s lambda, so suggest placing them inside there (as statics) scope them to the level that they're needed at.

Reply:
Fair point.